components/public-api-server/pkg/apiv1/workspace_test.go

```diff
@@ -662,7 +662,7 @@ var workspaceTestData = []workspaceTestDataEntry{
 				ContextUrl: "https://github.com/gitpod-io/gitpod",
 				Details: &v1.WorkspaceContext_Git_{
 					Git: &v1.WorkspaceContext_Git{
-						NormalizedContextUrl: "https://github.com/gitpod-io/gitpod",
+						NormalizedContextUrl: "https://github.com/gitpod-io/protocol.git",
```

Comment:
It's because input data is weird https://github.com/gitpod-io/gitpod/blob/7e0c605a3d470b8cee0e841e51da9b20022f4f4b/components/public-api-server/pkg/apiv1/workspace_test.go#L618

Reply:
Why would `https://github.com/gitpod-io/gitpod` normalize to `https://github.com/gitpod-io/protocol.git`?
Should we fix/change the original test?

Reply:
I have changed to `open-prebuild` prefix 🙏 . cc @svenefftinge